src/main/java/com/ioet/bpm/people/services/PasswordManagementService.java

```diff
@@ -0,0 +1,76 @@
+package com.ioet.bpm.people.services;
+
+import com.ioet.bpm.people.domain.PasswordHistory;
+import com.ioet.bpm.people.domain.Person;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+@Component
+public class PasswordManagementService {
+
+    private static final Logger log = LoggerFactory.getLogger(PasswordManagementService.class);
```

Comment:
You can remove this line and annotate the class with @Slf4j <- from lombok https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html